backend/api/Controllers/RobotController.cs

```diff
@@ -363,6 +362,16 @@ public async Task<ActionResult<IList<VideoStream>>> GetVideoStreams([FromRoute]
             return Conflict($"The Robot is not available ({robot.Status})");
         }
 
+        if (robot.Status is RobotStatus.Stuck)
```

Comment:
Can this ever be reached? The check above will return conflict for anything other than RobotStatus.Available

Reply:
Done - fixed